### ISSUE TITLE
Get rid of the user mentions for the leaderboard

### ISSUE THREAD

**Issue by jakesherman** [bticknor/elo-ranking-slackbot#27]
Certain users, specifically Dr. Wayne, have become annoyed at the Slack notifications that are triggered by the user-specific mentions when the leaderboard is displayed. Is there a way to fix this?

**Comment by bticknor** [comment]
Yup - right now the bot references users by @'ing their Slack user ID, which causes Slack to send notifications.  I presume that we could reference users by name instead, since `name` is a parameter of the User case class in the Slack client we are using:

https://github.com/slack-scala-client/slack-scala-client/blob/master/src/main/scala/slack/models/User.scala

Referencing users by name would prevent Slack from sending notifications, I believe.
